tests/excerptexport/test_downloads.py

```diff
@@ -12,14 +10,29 @@ def test_file_download(authenticated_client, user, output_file_with_file, output
     assert b''.join(response.streaming_content) == output_file_content
 
 
-def test_successful_file_attaching_changes_export_finished_timestamp(mocker, output_file_content, export):
+def test_successful_file_attaching_changes_export_finished_timestamp(mocker, some_fake_zip_file, export):
     margin = timedelta(minutes=1)
     now = datetime.now()
     mocker.patch.object(
-        ConversionApiClient, 'get_result_file',
-        side_effect=[ContentFile(output_file_content)],
+        ConversionApiClient, 'get_result_file_path',
+        side_effect=[some_fake_zip_file.name],
     )
     assert export.finished_at is None
     export._fetch_result_file()
     assert export.finished_at is not None
     assert (now - margin) < export.finished_at < (now + margin)
+
+
+def test_successful_file_attaching_removes_original_file(mocker, some_fake_zip_file, export):
+    mocker.patch.object(
+        ConversionApiClient, 'get_result_file_path',
+        side_effect=[some_fake_zip_file.name],
+    )
+    assert os.path.exists(some_fake_zip_file.name)
+    export._fetch_result_file()
+    assert bool(export.output_file.file)
```

Comment:
AFAIK, `assert` (like other control structures expecting a boolean context, such as `if` and `while`) already implicitly casts to `bool`. Is the explicit cast here for readability?